views/includes/check-your-answers/beneficial-owner-legal.html

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
maybe call this file 'legal-entity-trustee' so it is easier to tell what is used for and follows a similar naming to the 'individual-trustee' you created.

Reply:
Yep you are right my naming is not always the best 👍